.github/dependabot.yml

```diff
@@ -97,6 +97,16 @@ updates:
     schedule:
       interval: "weekly"
       day: "sunday"
+  -
+    package-ecosystem: "gomod"
+    directory: "/idgenerator/aws/xray"
```

Comment:
Do you think this package should be under here or is it ok to put it somewhere under propagators?

E.g. propagators/aws/xray/xrayidgenerator

Reply:
Would there be a situation where vendors will implement only an ID Generator but not propagator, or does anyone else have an opinion on this?

Reply:
Putting it under `propagators/aws/xray/xrayidgenerator` makes sense.  Do any other @open-telemetry/go-approvers have thoughts?

Reply:
I think starting with `propagators/aws/xray/idgenerator` is probably a good idea. Until there is a need to have only the ID generator imported having it be a separate module will be unnecessary overhead.

Reply:
Okay, I will move the ID Generator to be under `propagators/aws/xray/idgenerator`.